weblate/wladmin/views.py

```diff
@@ -77,6 +77,7 @@
     ("appearance", "manage-appearance", gettext_lazy("Appearance")),
     ("tools", "manage-tools", gettext_lazy("Tools")),
     ("machinery", "manage-machinery", gettext_lazy("Automatic suggestions")),
+    ("addons", "manage-addons", gettext_lazy("Addons")),
```

Comment:
Let's be consistent in naming:

```suggestion
    ("addons", "manage-addons", gettext_lazy("Add-ons")),
```